CRM/Contribute/Page/ContributionPage.php

```diff
@@ -247,21 +247,21 @@ public function &contributionLinks() {
           'name' => ts('Current Month-To-Date'),
           'title' => ts('Current Month-To-Date'),
           'url' => $urlString,
-          'qs' => "{$urlParams}&start={$monthDate}&end={$now}",
+          'qs' => "{$urlParams}&receive_date_low={$monthDate}&receive_date_high={$now}",
           'uniqueName' => 'current_month_to_date',
         ),
         CRM_Core_Action::REVERT => array(
           'name' => ts('Fiscal Year-To-Date'),
           'title' => ts('Fiscal Year-To-Date'),
           'url' => $urlString,
-          'qs' => "{$urlParams}&start={$yearDate}&end={$yearNow}",
+          'qs' => "{$urlParams}&receive_date_low={$yearDate}&recieve_date_high={$yearNow}",
```

Comment:
Typo in the "receive_date_high". Also down on line 264 the "receive_date_low".

Reply:
ug thanks

Reply:
thanks @demeritcowboy fixed @eileenmcnaughton